core/dbt/include/README.md

```diff
@@ -1 +1,10 @@
-# Include README
+# Include Module
+
+contains the a few key default `dbt-core` sub directories inlcuding.
+
+- `global_project` which defines the default implementations of jinja2 macros for `dbt-core ` which can be overwritten in each adapter repo to work more in line with those adapter plugins. to view adapter specific jinja2 changes please check the relevant adapter repo.
```

Comment:
Probably worth pointing to what you're referring to in at least one of the adapter repos with a note that each adapter is different so someone looking at this at least know where to go look.

Reply:
That's a good point will look for a macro that seems to be different from default in at least most of the adapters so it provides a varied look over.